packages/sdk/contractkit/README.md

```diff
@@ -46,13 +47,18 @@ await kit.web3.eth.getBalance(someAddress)
 ```ts
 import { newKit, CeloContract } from '@celo/contractkit'
 
-async function getKit(myAddress: string) {
-  const kit = newKit('https://alfajores-forno.celo-testnet.org:8545')
+async function getKit(myAddress: string, privateKey: string) {
+  const kit = newKit('https://alfajores-forno.celo-testnet.org')
 
-  // default from
+  // default from account 
   kit.defaultAccount = myAddress
+  
+  // add the account private key for tx signing when connecting to a remote node
+  kit.addAccount(privateKey)
```

Comment:
this works, but I would change to `kit.connection.addAccount` as that is the preferred method

Reply:
updated thanks